### ISSUE TITLE
Please make README.md a little more elaborate

### ISSUE THREAD

**Issue by luzpaz** [JiriValasek/Animate#2]
Hi @JiriValasek 
This looks awesome. Are you aware of the other workbenches that have similar functionality? How does this workbench differ?

Also can you add: 
- [ ] License info to README  
- [ ] Screenshots and/or short animated gifs  
- [ ] Installation instructions  
- [ ] a place on the forum that users/testers could give you feedback beside the issue queue of this repo
- [x] Add [topics](https://help.github.com/en/articles/classifying-your-repository-with-topics) (aka tags) to this repo like `freecad` `workbench` `addon` `animation` etc...

**Comment by JiriValasek** [comment]
Hi @luzpaz,
thanks, I've looked into several similar workbenches, but they usually missed some key component I need for my project. Also, some seemed quite complicated with few comments, little documentation.

I know about all the things you mentioned, they are on the list to be done after the code is finished or sometime before that. I wouldn't have gone public before finishing if I had not needed access to GitHub pages to be able to start working on documentation.

Anyway thanks for the feedback, I'll add those things in the near future.

**Comment by JiriValasek** [comment]
Hi, 
I finally finished all the requested things in your issue. What do you think of them?

**Comment by luzpaz** [comment]
Very cool. you did a lot of work there.  
some feedback:
1. The yellow headers and some of the usage of lighter colors on that page are hard to read with the white background. something should either change, the background or the color scheme.
2. There is an interest to provide an abridged version, of what you're in the website, on the README.md. This is because with your permission once the workbench is in a state where it can be used, we'd add it to the FreeCAD-addons repo. Which will allow this workbench to be easily downloaded through the FreeCAD Addon Manager. The Addon Manager utilizes thee README.md by displaying it as part of the way users can understand what it does.  
3. in your instructions in how to install your workbench you don't mention that on Linux/MacOS systems FreeCAD has a default `~/.FreeCAD/Mod` directory that can be used to download the workbench in to. 
4. I'm not sure you mention that it's important to stay up to date with you workbench and that users should expect to update the workbench periodically. (this is another reason why point 2, being added to the Addon manager, is convenient because it alerts users when you've updated the repository and a fast way for them to update.
5. You also don't mention the ability to use `git clone` as a convenient method to install the workbench and subsequently keeping up-to-date with it.

**Comment by luzpaz** [comment]
any more progress on this workbench?

**Comment by JiriValasek** [comment]
Not really... Unfortunately, I am kinda low on time to further develop this one.
